File: LAB_9_10/LAB10 20161811_20161442/LAB10 20161811_20161442/GameSoftLab10CS/GameSoft/GameSoftRRHH/RoleHistory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameSoftRRHH
{
    public class RoleHistory
    {
        private DateTime _startDate;
        private DateTime _endDate;
        private Role _role;
        private Employee _employee;
        private Department _department;

        public DateTime StartDate { get => _startDate; set => _startDate = value; }
        public DateTime EndDate { get => _endDate; set => _endDate = value; }
        public Role Role { get => _role; set => _role = value; }
        public Employee Employee { get => _employee; set => _employee = value; }
        public Department Department { get => _department; set => _department = value; }
    }
}

File: LAB_9_10/LAB10 20161811_20161442/LAB10 20161811_20161442/GameSoft-Netbeans/GameSoftRRHH/src/pucp/gamesoft/rrhh/model/RoleHistory.java
package pucp.gamesoft.rrhh.model;

import java.util.Date;
public class RoleHistory{
	private Date startDate;
	private Date endDate;
	private Role role;
	private Employee employee;
	private Department department;
	public RoleHistory(Department department, Employee employee, Date startDate, Date endDate){
		this.department = department;
		this.employee = employee;
		this.startDate = startDate;
		this.endDate = endDate;
	}
}